model/jpa/src/main/resources/META-INF/jpa-changelog-26.0.0.xml

```diff
@@ -93,6 +93,12 @@
     </changeSet>
 
     <changeSet author="keycloak" id="26.0.0-32583-drop-redundant-index-on-client-session">
+        <preConditions onSqlOutput="TEST" onFail="MARK_RAN">
+            <and>
+                <dbms type="oracle"/>
```

Comment:
The check for the DB should be removed, otherwise it wouldn't run on other databases.

```suggestion
```

Reply:
Thanks for the clarification. I just copied it from the 24.0.2 migration thinking this is necessary.